app/helpers.php

```diff
@@ -196,6 +196,15 @@ function read_image_properties_from_string($string)
     }
 }
 
+function require_login($text_key, $link_text_key)
```

Comment:
or just to make the link so `text_key` can have another parameters in addition to `link`?